docs/markdown/Python/python/python-third-party-dependencies.md

```diff
@@ -533,6 +521,104 @@ the user:
 indexes.add = ["http://$USERNAME:$PASSWORD@my.custom.repo/index"]
 ```
 
+### Local requirements
+
+There are two ways to specify local requirements from the filesystem:
+
+* [PEP 440 direct references](https://www.python.org/dev/peps/pep-0440/#direct-references)
+
+```python 3rdparty/python
+python_requirement(
+    name="django",
+    # Use an absolute path to a .whl or sdist file.
+    requirements=["Django @ file:///Users/pantsbuild/prebuilt_wheels/django-3.1.1-py3-none-any.whl"],
+)
+
+# Reminder: we could also put this requirement string in requirements.txt and use the
+# `python_requirements` target generator.
+```
+
+* The option `[python-repos].find_links`
+
+```toml pants.toml
+[python-repos]
+# Use an absolute path to a directory containing `.whl` and/or sdist files.
+find_links = ["file:///Users/pantsbuild/prebuilt_wheels"]
+```
+```shell
+❯ ls /Users/pantsbuild/prebuilt_wheels
+ansicolors-1.1.8-py2.py3-none-any.whl
+django-3.1.1-py3-none-any.whl
+```
+```python 3rdparty/BUILD
+# Use normal requirement strings, i.e. without file paths.
+python_requirement(name="ansicolors", requirements=["ansicolors==1.1.8"])
+python_requirement(name="django", requirements=["django>=3.1,<3.2"])
+
+# Reminder: we could also put these requirement strings in requirements.txt and use the
```

Comment:
Same here. We don't want to encourage people to use raw `python_requirement` targets.